duckduckgo/iOS
# Content Scope Scripts integration

Task/Issue URL: https://app.asana.com/0/72649045549333/1201631403895846/f
CC: @SlayterDev 

**Description**:
Enable content scope scripts integration so we do not have to embed js scripts within our app bundle.

**Steps to test this PR**:
1. Make sure all tests are passing.
2. Retest fingerprinting protection.
3. Make sure canvas fingerprinting works.
4. Make sure gpc feature works.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
